tests/reproducibility/test_attri2vec.py

```diff
@@ -45,18 +45,16 @@ def unsup_attri2vec_model(generator, optimizer, bias, normalize):
     return model
 
 
-def unsup_attri2vec(
-    g,
-    optimizer,
-    batch_size=4,
-    epochs=4,
-    bias=True,
-    normalize="l2",
-    number_of_walks=1,
-    walk_length=5,
-    seed=0,
-    shuffle=True,
-):
+def unsup_attri2vec(g, batch_size, shuffle):
+
+    epochs = 4
+    bias = True
+    normalize = "l2"
+    seed = 0
+    optimizer = tf.optimizers.Adam(1e-3)
+    number_of_walks = 1
```

Comment:
Oh, should we do 

```suggestion
    number_of_walks = 2
```

or higher, to catch any problems that only manifest across a transition (i.e. not when there's only one example)?